dotCMS/src/main/java/com/dotmarketing/fixtask/tasks/FixTask00095DeleteOrphanRelationships.java

```diff
@@ -0,0 +1,173 @@
+                FixAssetsProcessStatus
+                        .setDescription("task 95: DeleteOrphanRelationships");
+                DotConnect dc = new DotConnect();
+                dc.setSQL(VERIFICATION_QUERY);
+                modifiedData = dc.loadResults();
+                total = modifiedData != null ? modifiedData.size() : 0;
+                FixAssetsProcessStatus.setTotal(total);
+                getModifiedData();
+                if (total > 0) {
+                    try {
+                        HibernateUtil.startTransaction();
+                        cleanUpOrphanRelationships();
+                        
+                        HibernateUtil.commitTransaction();
+                        // Set the number of records that were fixed
+                        FixAssetsProcessStatus.setErrorsFixed(modifiedData.size());
+                    } catch (Exception e) {
+                        Logger.error(
+                                this,
+                                "Unable to clean orphaned content type fields.",
+                                e);
+                        HibernateUtil.rollbackTransaction();
+                        modifiedData.clear();
+                    }
+                    finally {
+                        flushCacheRegions();
+                    }
+                    FixAudit Audit = new FixAudit();
+                    Audit.setTableName("relationship");
+                    Audit.setDatetime(new Date());
+                    Audit.setRecordsAltered(total);
+                    Audit.setAction("task 95: DeleteOrphanRelationships");
+                    HibernateUtil.save(Audit);
+                    HibernateUtil.commitTransaction();
+                    returnValue.add(FixAssetsProcessStatus.getFixAssetsMap());
+                    Logger.debug(
+                            FixTask00095DeleteOrphanRelationships.class,
+                            "Ending DeleteOrphanRelationships");
+                }
+            } catch (Exception e) {
+                Logger.debug(
+                        FixTask00095DeleteOrphanRelationships.class,
+                        "There was a problem during DeleteOrphanRelationships",
+                        e);
+                HibernateUtil.rollbackTransaction();
+                FixAssetsProcessStatus.setActual(-1);
+            } finally {
+                FixAssetsProcessStatus.stopProgress();
+            }
+        }
+        return returnValue;
+    }
+
+    @Override
+    public List<Map<String, String>> getModifiedData() {
+        if (modifiedData.size() > 0) {
+            XStream _xstream = new XStream(new DomDriver());
+            Date date = new Date();
+            SimpleDateFormat sdf = new SimpleDateFormat("dd-MM-yyyy_HH-mm-ss");
+            String lastmoddate = sdf.format(date);
+            File _writing = null;
+            if (!new File(ConfigUtils.getBackupPath() + File.separator
+                    + "fixes").exists()) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "fixes" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)